client/me/help/chat-closure-notice/index.jsx

```diff
@@ -0,0 +1,74 @@
+/**
+ * Fixed notice about upcoming support closures
+ */
+
+/**
+ * External dependencies
+ */
+import React, { Component } from 'react';
+import i18n, { localize } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import FormSectionHeading from 'components/forms/form-section-heading';
+import olarkStore from 'lib/olark-store';
+import { title, upcoming, closed } from './messages';
+
+const Notice = localize( ( { translate, closedFrom, closedTo, reason } ) =>
+	<div className="chat-closure-notice">
+		<FormSectionHeading>{ title( { translate, closedFrom, closedTo, reason } ) }</FormSectionHeading>
```

Comment:
For this and `closed`, I'm passing these in case they might be used in messages in the future